homeassistant/components/google/manifest.json

```diff
@@ -4,11 +4,7 @@
   "config_flow": true,
   "dependencies": ["auth"],
   "documentation": "https://www.home-assistant.io/integrations/calendar.google/",
-  "requirements": [
-    "google-api-python-client==2.38.0",
-    "httplib2==0.20.4",
-    "oauth2client==4.1.3"
-  ],
+  "requirements": ["gcal-sync==0.4.1", "oauth2client==4.1.3"],
```

Comment:
The library is missing a link to the github source on PyPI:
https://pypi.org/project/gcal-sync/

Reply:
It's also missing production requirements (aiohttp, pydantic) needed.

https://github.com/allenporter/gcal_sync/blob/f97517033d2235b4950b96b6e6fbf5c253d80d18/setup.cfg#L16

Reply:
Probably should also set a minimum Python version to 3.9 as `zoneinfo` is imported.

Reply:
Fixed. Isee gaps in other projects i maintain and will also update those also.